README.md

```diff
@@ -70,6 +70,22 @@ public class MyActivity extends NaviActivity {
 }
 ```
 
+If you want some kotlin goodness, you can use bind() syntax
+
+```java
+myObservable
+    .bindToLifecycle(myView)
+    .subscribe { }
+
+```
+
+```java
+myObservable
```

Comment:
Nah. I was just saying that this is two separate code samples but could just be wrapped into one.